operator/pkg/ciliumendpointslice/manager.go

```diff
@@ -408,6 +410,8 @@ func (c *cesMgr) getCEPCountInCES(cesName string) (cnt int) {
 		ces.backendMutex.RLock()
 		cnt = len(ces.ces.Endpoints)
 		ces.backendMutex.RUnlock()
+	} else {
+		log.Infof("Attempted to getCEPCountInCES non-existent CES : %v, skipping.", cesName)
```

Comment:
Hmm, is this an info level log message or debug? I am having a bit of a struggle understanding how a user could interpret this message and take action if they see it in their cluster.